app/views/authentication_mailer/sign_in_email.text.erb

```diff
@@ -2,6 +2,11 @@
   Dear <%= @application_form.first_name %>
 <% end %>
 
-Sign in to continue your application for teacher training:
+<% if FeatureFlag.active?(:continuous_applications) %>
+  Sign in to continue applying for teacher training:
```

Comment:
This doesn't sound right to me. "Continue applying" makes me think of "keep going". Wouldn't something like "continue the process of applying" work better?